packages/editor/src/lib/exports/getSvgJsx.tsx

```diff
@@ -177,8 +180,20 @@ function SvgExport({
 			isDarkMode,
 			waitUntil,
 			addExportDef,
+			scale,
+			pixelRatio,
+			async resolveAssetUrl(assetId, width) {
+				const asset = editor.getAsset(assetId)
+				if (!asset || asset.type === 'bookmark') return null
```

Comment:
this asset.type bookmark check seems brittle. one day when this TLAudioAsset PR finally lands 🙄 https://github.com/tldraw/tldraw/pull/4290 it'll also break this check (i think). maybe we should have some propery that denotes whether it is exportable like this.

Reply:
yeah... this is a bit of an issue with our exports as a whole at the moment. although the audio asset at least has a `src` property? But yeah i wonder if assets are even quite the right abstraction here :/

I'll invert this to explicitly check for image/video